base/dft.jl

```diff
@@ -264,7 +264,7 @@ A_mul_B!(y::AbstractArray, p::ScaledPlan, x::AbstractArray) =
 # or odd).
 
 for f in (:brfft, :irfft)
-    pf = symbol(string("plan_", f))
+    pf = Symbol(string("plan_", f))
```

Comment:
Note that usages like this should probably be replaced with `Symbol("plan_", f)`.